CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Changed
+- The event store persists the event `causation_id`. To facilitate this
+  a `causation_id` column has been added to the `events` table and the
+  `write_events` function has been altered. Event Sourcery apps will need
+  to ensure these DB changes have been applied to use this version of
+  Event Sourcery.
```

Comment:
I personally like the screaming post install messages we use in double entry. Some specific instructions on how to do it would also be a good bonus.